frontend/src/stores/redux/user/slice.ts

```diff
@@ -0,0 +1,35 @@
+import { createSlice, PayloadAction } from '@reduxjs/toolkit'
+import {RootState} from "../store";
+import {ILoggedUserState} from "./types";
+import {UserType} from "./loggedUser";
+
+const initialState: ILoggedUserState = {
+    loggedUserState: {
```

Comment:
Initial state cannot have real values. All fields must be empty or null. Id = -1